fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java

```diff
@@ -622,16 +622,14 @@ public void validateHasPermissionTo(final String function, final List<String> al
     public void validateHasPermissionTo(final String function) {
         if (hasNotPermissionTo(function)) {
             final String authorizationMessage = "User has no authority to: " + function;
-            LOG.info("Unauthorized access: userId: {} action: {} allowed: {}", getId(), function, getAuthorities());
+            LOG.info("Unauthorized access: userId: {} action: {} allowed: {}", getId(), function.replaceAll("[\n\r\t]", "_"),
```

Comment:
I'm getting the feeling now that there should be a utility class for escaping log parameters.
Do you mind creating a util and reusing it?